tests/state/ViewerState/time_slice.test.ts

```diff
@@ -117,11 +119,59 @@ describe("useViewerStateStore: TimeSlice", () => {
     await act(async () => {
       await result.current.setFrame(3);
     });
-    expect(result.current.currentFrame).toBe(3);
     expect(result.current.pendingFrame).toBe(3);
 
     await clearDatasetAsync(result);
-    expect(result.current.currentFrame).toBe(0);
     expect(result.current.pendingFrame).toBe(0);
   });
+
+  describe("serializeTimeSlice", () => {
+    it("serializes 0 values", async () => {
+      const { result } = renderHook(() => useViewerStateStore());
+      await act(async () => {
+        await result.current.setFrame(0);
+      });
+      expect(serializeTimeSlice(result.current)[UrlParam.TIME]).toBe("0");
+    });
+
+    it("serializes time value", async () => {
+      const { result } = renderHook(() => useViewerStateStore());
```

Comment:
nit: 
`const frameNumber = 155`
`setFrame(frameNumber)`
and then on like 142: `toBe(frameNumber.toString())`

Reply:
Fixed here! I'll update the time fix PR with more variables too since it also touches the same test file.